README.md

```diff
@@ -4,60 +4,150 @@
+project to issue tokens, called Grain, to contributors based on their Cred.
+Grain is purchased by sponsors of the project, and gives sponsors the ability
+to influence Cred scores.
 
-Cred is computed by constructing a Cred Graph showing every contribution to a project, and then running a version of the PageRank
-algorithm to assign scores to every contribution and contributor.
+You can read more at [sourcecred.io].
 
-We currently support loading data from GitHub, Discourse, Discord, and via custom "initiatives".
+[sourcecred.io]: https://sourcecred.io/
 
-Please check out [our website] for more information. If you'd like to get involved as a contributor, please drop by [our Discord]
-and say "hi"!
+## Plugin Architecture
 
-[our website]: https://sourcecred.io/
-[our Discord]: https://sourcecred.io/discord
+SourceCred is organized around a plugin architecture, which ensures that it can
+track and reward any kind of contribution, so long as you can assign addresses
+to contributions, and record how they relate to one another. Currently, we have the following four plugins:
+
+- **GitHub**: Loads GitHub repositories, including issues, pull requests, and reviews
+- **Discourse**: Loads Discourse forums, including posts, topics, and likes
+- **Discord**: Loads Discord servers, including messages and reactions
+- **Initiatives**: Loads manually added contributions. Still in alpha.
+
+Every plugin has a two-part name in the form `$OWNER/$NAME`; for example,
+SourceCred's own GitHub plugin is named `sourcecred/github`.
+
+## The Instance System
+
+SourceCred is organized around the concept of "instances". A SourceCred instance
+contains all of the configuration and data associataed with Cred and Grain, and
+optionally may be set up as a deployable website that displays those scores.
+Each instance has the following directory structure:
+
+```
+./sourcecred.json   # Lists enabled plugins
+./config            # User-edited config files
+./data              # Persistent data, e.g. ledger history
+./output            # Output data, may be removed/regenerated
+./site              # Bundled frontend, if included
+./cache             # Temporary data, should not be checked in to git
+```
+
+We recommend storing instances in a Git repository. The best way to set up an
+instance is by forking [sourcecred/example-instance].
 
-## Current Status
+Once your instance is setup, you can update it with the following commands:
 
-SourceCred is still in beta; as such, the interfaces are in flux and the documentation is spotty.
-We're working on a polished release which will include more documentation, and more maintainable instances. We expect this to land
-by early July.
+- `sourcecred load`: Regenerate the cache
+- `sourcecred graph`: Recompute graphs from cache
+- `sourcecred score`: Re-run Cred calculations
+- `sourcecred site`: Regenerate the website (potentially upgrading it)
+- `sourcecred go`: Runs `load`, `graph` and `score` in sequence.
```

Comment:
Should be `yarn sourcecred <command>` instead. Or `yarn <command>` if its in package.json scripts already

Reply:
Changed to `yarn command` format.